src/button/tests/unit/Button.ts

```diff
@@ -1,12 +1,13 @@
+
```

Comment:
why the new line?

Reply:
Formatting failure. Removed these from all the files they were in.